packages/compass-shell/src/components/compass-shell/compass-shell.jsx

```diff
@@ -136,6 +138,13 @@ export class CompassShell extends Component {
     );
   }
 
+  hideInfoModal() {
+    this.setState({ showInfoModal: false });
+    if (this.shellRef.current) {
```

Comment:
Is this check necessary? How would you get here if current is falsey?

Reply:
the current can be null until the `Shell` component has rendered.